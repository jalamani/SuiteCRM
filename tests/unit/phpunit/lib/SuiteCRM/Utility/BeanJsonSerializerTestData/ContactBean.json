{
  "id": "00000000-0000-0000-0000-000000000000",
  "module_name": "Contacts",
  "module_dir": "Contacts",
  "date_entered": "2018-06-12 11:01:34",
  "date_modified": "2018-06-12 11:01:34",
  "modified_user_id": "1",
  "created_by": "1",
  "description": null,
  "deleted": "0",
  "assigned_user_id": "seed_max_id",
  "salutation": "Ms.",
  "first_name": "Cindy",
  "last_name": "Espana",
  "title": "Director Operations",
  "photo": null,
  "department": "Genetics",
  "do_not_call": "0",
  "phone_home": "(445) 163-3872",
  "phone_mobile": "(978) 836-3300",
  "phone_work": "(711) 112-3512",
  "phone_other": null,
  "phone_fax": null,
  "lawful_basis": null,
  "date_reviewed": null,
  "lawful_basis_source": null,
  "primary_address_street": "345 Sugar Blvd.",
  "primary_address_city": "Salt Lake City",
  "primary_address_state": "CA",
  "primary_address_postalcode": "58821",
  "primary_address_country": "USA",
  "alt_address_street": null,
  "alt_address_city": null,
  "alt_address_state": null,
  "alt_address_postalcode": null,
  "alt_address_country": null,
  "assistant": null,
  "assistant_phone": null,
  "lead_source": "Web Site",
  "reports_to_id": null,
  "birthdate": null,
  "campaign_id": null,
  "joomla_account_id": null,
  "portal_account_disabled": null,
  "portal_user_type": "Single",
  "age": null,
  "id_c": null,
  "jjwg_maps_lng_c": null,
  "jjwg_maps_lat_c": null,
  "jjwg_maps_geocode_status_c": null,
  "jjwg_maps_address_c": null,
  "email1": "kid79@example.co.jp",
  "modified_by_name": "Administrator",
  "created_by_name": "Administrator",
  "assigned_user_name": "Max Jensen",
  "account_name": "Start Over Trust",
  "account_id": "39363e96-5eed-0221-9889-5b1fa86ebb52",
  "opportunity_role_fields": "",
  "report_to_name": "",
  "campaign_name": "",
  "c_accept_status_fields": "",
  "m_accept_status_fields": "",
  "e_invite_status_fields": "",
  "e_accept_status_fields": "",
  "column_fields": [
    "id",
    "name",
    "date_entered",
    "date_modified",
    "modified_user_id",
    "modified_by_name",
    "created_by",
    "created_by_name",
    "description",
    "deleted",
    "created_by_link",
    "modified_user_link",
    "assigned_user_id",
    "assigned_user_name",
    "assigned_user_link",
    "SecurityGroups",
    "salutation",
    "first_name",
    "last_name",
    "full_name",
    "title",
    "photo",
    "department",
    "do_not_call",
    "phone_home",
    "email",
    "phone_mobile",
    "phone_work",
    "phone_other",
    "phone_fax",
    "email1",
    "email2",
    "invalid_email",
    "email_opt_out",
    "lawful_basis",
    "date_reviewed",
    "lawful_basis_source",
    "primary_address_street",
    "primary_address_street_2",
    "primary_address_street_3",
    "primary_address_city",
    "primary_address_state",
    "primary_address_postalcode",
    "primary_address_country",
    "alt_address_street",
    "alt_address_street_2",
    "alt_address_street_3",
    "alt_address_city",
    "alt_address_state",
    "alt_address_postalcode",
    "alt_address_country",
    "assistant",
    "assistant_phone",
    "email_addresses_primary",
    "email_addresses",
    "email_addresses_non_primary",
    "email_and_name1",
    "lead_source",
    "account_name",
    "account_id",
    "opportunity_role_fields",
    "opportunity_role_id",
    "opportunity_role",
    "reports_to_id",
    "report_to_name",
    "birthdate",
    "accounts",
    "reports_to_link",
    "opportunities",
    "bugs",
    "calls",
    "cases",
    "direct_reports",
    "emails",
    "documents",
    "leads",
    "meetings",
    "notes",
    "project",
    "project_resource",
    "am_projecttemplates_resources",
    "am_projecttemplates_contacts_1",
    "tasks",
    "tasks_parent",
    "notes_parent",
    "user_sync",
    "campaign_id",
    "campaign_name",
    "campaigns",
    "campaign_contacts",
    "c_accept_status_fields",
    "m_accept_status_fields",
    "accept_status_id",
    "accept_status_name",
    "prospect_lists",
    "sync_contact",
    "fp_events_contacts",
    "aos_quotes",
    "aos_invoices",
    "aos_contracts",
    "e_invite_status_fields",
    "event_status_name",
    "event_invite_id",
    "e_accept_status_fields",
    "event_accept_status",
    "event_status_id",
    "project_contacts_1",
    "aop_case_updates",
    "joomla_account_id",
    "portal_account_disabled",
    "joomla_account_access",
    "portal_user_type",
    "age",
    "jjwg_maps_lng_c",
    "jjwg_maps_lat_c",
    "jjwg_maps_geocode_status_c",
    "jjwg_maps_address_c"
  ],
  "fetched_row": {
    "id": "00000000-0000-0000-0000-000000000000",
    "date_entered": "2018-06-12 11:01:34",
    "date_modified": "2018-06-12 11:01:34",
    "modified_user_id": 1,
    "created_by": 1,
    "description": null,
    "deleted": 0,
    "assigned_user_id": "seed_max_id",
    "salutation": "Ms.",
    "first_name": "Cindy",
    "last_name": "Espana",
    "title": "Director Operations",
    "photo": null,
    "department": "Genetics",
    "do_not_call": 0,
    "phone_home": "(445) 163-3872",
    "phone_mobile": "(978) 836-3300",
    "phone_work": "(711) 112-3512",
    "phone_other": null,
    "phone_fax": null,
    "lawful_basis": null,
    "date_reviewed": null,
    "lawful_basis_source": null,
    "primary_address_street": "345 Sugar Blvd.",
    "primary_address_city": "Salt Lake City",
    "primary_address_state": "CA",
    "primary_address_postalcode": 58821,
    "primary_address_country": "USA",
    "alt_address_street": null,
    "alt_address_city": null,
    "alt_address_state": null,
    "alt_address_postalcode": null,
    "alt_address_country": null,
    "assistant": null,
    "assistant_phone": null,
    "lead_source": "Web Site",
    "reports_to_id": null,
    "birthdate": null,
    "campaign_id": null,
    "joomla_account_id": null,
    "portal_account_disabled": null,
    "portal_user_type": "Single",
    "age": null,
    "id_c": null,
    "jjwg_maps_lng_c": null,
    "jjwg_maps_lat_c": null,
    "jjwg_maps_geocode_status_c": null,
    "jjwg_maps_address_c": null,
    "email1": "kid79@example.co.jp"
  },
  "fetched_rel_row": {
    "modified_by_name": "Administrator",
    "created_by_name": "Administrator",
    "assigned_user_name": "Max Jensen",
    "account_name": "Start Over Trust",
    "account_id": "39363e96-5eed-0221-9889-5b1fa86ebb52",
    "opportunity_role_fields": "",
    "report_to_name": "",
    "campaign_name": "",
    "c_accept_status_fields": "",
    "m_accept_status_fields": "",
    "e_invite_status_fields": "",
    "e_accept_status_fields": ""
  }
}