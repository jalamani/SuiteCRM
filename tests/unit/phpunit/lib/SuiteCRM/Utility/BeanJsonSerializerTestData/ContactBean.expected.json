{
  "id": "00000000-0000-0000-0000-000000000000",
  "meta": {
    "created": {
      "date": "2018-06-12 11:01:34",
      "user_id": 1,
      "user_name": "Administrator"
    },
    "modified": {
      "date": "2018-06-12 11:01:34",
      "user_id": 1,
      "user_name": "Administrator"
    },
    "assigned": {
      "user_id": "seed_max_id",
      "user_name": "Max Jensen"
    }
  },
  "name": {
    "salutation": "Ms.",
    "first": "Cindy",
    "last": "Espana"
  },
  "account": {
    "title": "Director Operations",
    "department": "Genetics",
    "id": "39363e96-5eed-0221-9889-5b1fa86ebb52",
    "name": "Start Over Trust"
  },
  "phone": {
    "home": "(445) 163-3872",
    "mobile": "(978) 836-3300",
    "work": "(711) 112-3512",
    "other": null,
    "fax": null
  },
  "address": {
    "primary": {
      "street": "345 Sugar Blvd.",
      "city": "Salt Lake City",
      "state": "CA",
      "postalcode": 58821,
      "country": "USA"
    },
    "alt": {
      "street": null,
      "city": null,
      "state": null,
      "postalcode": null,
      "country": null
    }
  },
  "assistant": {
    "name": null,
    "phone": null
  },
  "reports_to": {
    "id": null,
    "name": ""
  },
  "campaign": {
    "name": "",
    "id": null
  },
  "lead": {
    "type": "Web Site"
  },
  "birthdate": null,
  "age": null,
  "email": [
    "kid79@example.co.jp"
  ],
  "description": ""
}